tsconfig.json

```diff
@@ -8,8 +8,5 @@
         "outDir": "_out",
         "forceConsistentCasingInFileNames": true
     },
-    "files": [
-        "index.ts",
-        "auth.ts"
-    ]
+    "include": ["*.ts"]
```

Comment:
I think this was missing `interface.ts` and `proxy.ts` when compiling

Reply:
nope.  they compiled because index and auth referenced interface and proxy.

Reply:
I reverted this change